300-children-exercises/array-exercises/ex3.rb

```diff
@@ -0,0 +1,11 @@
+=begin
+tim vi tri cac so nho nhat trong mang
+=end
+arr = 10.times.map{rand(-10..10)}
+min = arr[0] 
+min_index = []
+arr.each {|a| min > a ? a = min : nil} 
```

Comment:
cái chỗ này thì if 1 ngôi thôi, 2 ngồi thêm nil thì thừa

Reply:
xoa nil di thi no bao loi. 
ex3.rb:7: syntax error, unexpected '}', expecting ':'
...rr.each {|a| min > a ? a = min} 
ex3.rb:11: syntax error, unexpected end-of-input, expecting '}'

Reply:
nó báo lỗi đúng rồi, ? : là dùng cho if else 2 ngôi, còn xóa nil đi thì là if 1 ngôi thôi, thì sửa thành `a=min if min > a`